### ISSUE TITLE
Hard dependency on Celery?

### ISSUE THREAD

**Issue by Bartvds** [marcosgabarda/django-gum#5]
It looks like the Celery dependency is not used for core functionality but only to convenience decorate some methods? https://github.com/marcosgabarda/django-gum/search?utf8=%E2%9C%93&q=celery

Maybe this dependency should be dropped and the Celery integration made optional? It's cool to have some helpers ready for those who need it but not everyone is on the Celery train just yet.

**Comment by marcosgabarda** [comment]
You're right! I always use Celery, but it's true that this integration should be optional. I'll work on this!

Thanks for the idea! :)
